code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -47,3 +47,34 @@
 	embedding = list(embed_chance=100, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	wound_falloff_tile = -5
 	shrapnel_type = /obj/item/ammo_casing/harpoon
+
+// Rebar (Rebar Crossbow)
+/obj/projectile/bullet/rebar
+	name = "rebar"
+	icon_state = "rebar"
+	damage = 40
+	stamina = 20
```

Comment:
In any instance where actual damage is higher than stamina damage, stamina damage is practically meaningless. The stamina var is still reduced by most forms of armor.

Reply:
And, I'll be honest, 40 damage is a lot of power to just be able to produce out of iron rods. I'd honestly consider winding this back. If we use the improv shell as a metric, somewhere closer to the 30 damage range is maybe better.

Reply:
Noted. Winding it back.